frappe/utils/pdf.py

```diff
@@ -16,8 +16,7 @@ def get_pdf(html, options=None, output = None):
 
 	options.update({
 		"disable-javascript": "",
-		"disable-local-file-access": "",
-		"disable-smart-shrinking": ""
```

Comment:
- `disable-smart-shrinking` was added to restore the font size as it was visibly smaller due to `smart-shrinking`.
- Add a checkbox `disable-smart-shrinking` in Print Format, and if checked, add `disable-smart-shrinking` to the dict. Else this change will break print previews for other users.